### DISCUSSION TITLE
Docker: SSL cerfificates Issues with connecting to postgresql database

### DISCUSSION THREAD

**Discussion by durantsmedia** [lightdash/lightdash#6476] [General]
I am new to Docker & lightdash.

I am getting issues connecting to an external database I get the following error "Error running postgres query: Error: self signed certificate in certificate chain"

I am unsure how to update the certificates using Docker Compose.

**Comment by owlas** [comment]
Hey @durantsmedia where are you hosting your postgres database is it running locally on your machine?

**Reply by durantsmedia** [reply]
hello @owlas,

I am running it on a self-hosted server. I have tested with two independently managed Postgresql instances, and I get the same errors. When I use the development environment on my machine (using dbt or lightdash dbt commands), it connects to the same database server with no errors. I suspect it is because the database uses letsencrypt SSL certificates.

**Comment by jboilesen** [comment]
Facing this same issue here: local dbt, local postgresl, docker-compose lightdash installation

-- UPDATE --
This workaround works: https://github.com/lightdash/lightdash/discussions/6520#discussioncomment-6597443
